### PR TITLE
Ubuntu Touch 6.0.0_r1

### PR DESCRIPTION
**Pull request by vikoadi**

### CODE REVIEW

File: compress.bash [hunk 1]
```diff
@@ -11,19 +11,19 @@
 # Name of the ROM. No Spaces Please.
 # Example: CyanogenMod
 
-name=ColorOS
+name=Ubuntu Touch 6.0.0_r1
```
Comment (msfjarvis) [inline compress.bash]: Spaces will break the script. Make the name `UbuntuTouch` 
@regalstreak, remind me tomorrow to come up with a fix for this.

Comment (regalstreak) [reply compress.bash]: Sure

**Comment by vikoadi** [comment]
I'm still not very sure about putting version in the name, I guess no versioning is the way to go

**Comment by msfjarvis** [comment]
People, please enable "Allow Edits by maintainers" so that we can fix stuff online and merge your PRs

**Pull request closed without merging**